javascript/src/controllers/file-size-limit-controller.js

```diff
@@ -0,0 +1,35 @@
+import { Controller } from '@hotwired/stimulus'
+
+export class FileSizeLimitController extends Controller {
+  static targets = ['trixEditor']
+  static values = { limit: Number, message: String }
+
+  connect () {
+    this.trixEditorTarget.addEventListener(
+      'trix-file-accept',
+      this.checkLimit.bind(this)
+    )
+  }
+
+  disconnect () {
+    this.trixEditorTarget.removeEventListener(
+      'trix-file-accept',
+      this.checkLimit
```

Comment:
esto no va a funcionar, porque `this.checkLimit` es una diferente funcion a la que le pasaste en el `connect` con `this.checkLimit.bind(this)`. Al agregarle el `.bind(this)` eso efectivamente esta creando una nueva funcion

Reply:
con la sugerencia en la línea 8, esto ya funcionaría.